### PR TITLE
Akward Sitting Fixes Part 2

### LINKED ISSUE #1796: Akward Sitting Positions Part 2
@jonpas 
#1541

Arma 3 Dev: 909fbb5dc4309b15f6c34ca78917f1ffec4b80e3

**Mods:**
* @cba_a3
* @ace3

**Description:**
When sitting down on chairs positioned in a building the body clips through the ground.
ASL ATL problem?

image

**Steps to reproduce:**
* Grab a chair placed in a building
* Sit down

### PR DESCRIPTION
**Pull request by jonpas**
Fixes #1796 

Read https://github.com/acemod/ACE3/issues/1795#issuecomment-118667712 for Part 1.

### CODE REVIEW

File: addons/sitting/XEH_clientInit.sqf [hunk 1]
```diff
@@ -7,6 +7,6 @@ if !(hasInterface) exitWith {};
 ["isNotSitting", {!((_this select 0) getVariable [QGVAR(isSitting), false])}] call EFUNC(common,addCanInteractWithCondition);
 
 // Handle interruptions
-["medical_onUnconscious", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
-["SetHandcuffed", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
-["SetSurrendered", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
+["medical_onUnconscious", {_player call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
```
Comment (PabstMirror) [inline addons/sitting/XEH_clientInit.sqf]: Throwing:
 ```
Error position: <_player call ace_sitting_fnc_handleInter>
Error Undefined variable in expression: _player
File z\ace\addons\sitting\XEH_clientInit.sqf, line 10
```

Test codes:
```
[] spawn {
    [player, true] call ace_medical_fnc_setUnconscious;
    sleep 10;
    [player, false] call ace_medical_fnc_setUnconscious;
};

[] spawn {
    ["SetHandcuffed", [player, true]] call ace_common_fnc_localEvent;
    sleep 10;
    ["SetHandcuffed", [player, false]] call ace_common_fnc_localEvent;
};
```


I don't think this should be changed at all and keep it as `_this`.  
For all 3 events the unit is the first argument, which is what handleInterrupt is expecting.

Comment (jonpas) [inline addons/sitting/XEH_clientInit.sqf]: Weird, worked just fine for me, I thought I'd remove number of arguments sending. On that note, are there any performance differences in how many arguments you are sending, even if so small it's not worth it?

**Comment by jonpas** [comment]
Also removed `SetSurrendered` as that option is not available when sitting.

**Comment by jonpas** [comment]
This is ready to be merged from my side. I'll work on other fixes in a separate PR (time to learn about CfgMoves).

**Pull request merged**